packages/config/config/devices/0x039a/doorWindow.json

```diff
@@ -0,0 +1,91 @@
+{
+	"manufacturer": "Inteset",
+	"manufacturerId": "0x039a",
+	"label": "Inteset Door/Window Sensor",
+	"description": "Door/Window Sensor",
```

Comment:
@blhoward2 what should we do about these?

Reply:
I've had a few like this and I searched for the actual model number or part number.

Reply:
Found the model number from what I can tell.  I have since included it.